addon/services/cookies.js

```diff
@@ -31,6 +31,8 @@ export default Ember.Service.extend({
     return A(all).reduce((acc, cookie) => {
       if (!isEmpty(cookie)) {
         let [key, value] = cookie.split('=');
+        key   = key   || '';
```

Comment:
I'm not sure we should even add the cookie to the returned hash when the key is empty. Is a cookie with an empty name actually allowed according to the cookie spec?